py-polars/docs/source/reference/series/index.rst

```diff
@@ -20,7 +20,6 @@ This page gives an overview of all public Series methods.
    list
    modify_select
    miscellaneous
-   plot
```

Comment:
I'm removing Series.plot as Altair expects 2D structures (and in any case, I think the majority of use cases would be 2D plots anyway? if not, it's easy to call `to_frame()`)

Reply:
Could we just call `to_frame` internally there? It's not too odd to think someone might want to see a quick histogram from some Series values 🤔